.github/workflows/build.yml

```diff
@@ -0,0 +1,64 @@
+name: "Build"
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request: ~
+  release:
+    types: ["created"]
+  schedule:
+    -
+      cron: "0 1 * * 6" # Run at 1am every Saturday
+
+jobs:
+  build:
+    runs-on: "ubuntu-latest"
+
```

Comment:
You can define a clearer name using the below versions from matrix for each test